### PR TITLE
add FluidSynth gain setting to config

### PR DESCRIPTION
**Pull request by rfomin**
Increase volume for "silent" soundfonts [[doomworld]](https://www.doomworld.com/forum/topic/129928-new-sc55-soundfont-266mb-all-new-441k-samples/?do=findComment&comment=2522430) Although Trevor0402's SC-55 SoundFont.v1.2b volume levels are fine for me.

### CODE REVIEW

File: src/i_flmusic.c [hunk 1]
```diff
@@ -168,7 +169,8 @@ static boolean I_FL_InitMusic(void)
 static void I_FL_SetMusicVolume(int volume)
 {
     // FluidSynth's default is 0.2. Make 1.0 the maximum.
-    fluid_synth_set_gain(synth, (float)volume / 15);
+    // 0 -- 0.2 -- 10.0
+    fluid_synth_set_gain(synth, ((float)volume / 15) * (float)mus_gain);
```
Comment (fabiangreffrath) [inline src/i_flmusic.c:173]: Isn't this a bit coarse? I mean the next step would be `mus_gain = 2` which would double up the volume and the final step would amplify by factor 10?

Comment (rfomin) [reply src/i_flmusic.c:173]: How about this:
```diff
diff --git a/src/i_flmusic.c b/src/i_flmusic.c
index eb21843..6885b22 100644
--- a/src/i_flmusic.c
+++ b/src/i_flmusic.c
@@ -170,7 +170,7 @@ static void I_FL_SetMusicVolume(int volume)
 {
     // FluidSynth's default is 0.2. Make 1.0 the maximum.
     // 0 -- 0.2 -- 10.0
-    fluid_synth_set_gain(synth, ((float)volume / 15) * (float)mus_gain);
+    fluid_synth_set_gain(synth, ((float)volume / 15) * ((float)mus_gain / 10));
 }

 static void I_FL_PauseSong(void *handle)
diff --git a/src/m_misc.c b/src/m_misc.c
index bbc32f1..90a0cc9 100644
--- a/src/m_misc.c
+++ b/src/m_misc.c
@@ -2204,7 +2204,7 @@ default_t defaults[] = {
   {
     "mus_gain",
     (config_t *) &mus_gain, NULL,
-    {1}, {1, 10}, number, ss_none, wad_no,
+    {10}, {1, 100}, number, ss_none, wad_no,
     "amplify FluidSynth output level"
   },
 #endif
```

Comment (fabiangreffrath) [reply src/i_flmusic.c:173]: Looks better, but let's be consistent and use percents like Pr+:
https://github.com/coelckers/prboom-plus/blob/d7f95371d59e6889c61f02aea65e2d28a9f2b2a5/prboom2/src/MUSIC/flplayer.c#L167

Comment (rfomin) [reply src/i_flmusic.c:173]: So it will be 0-1000%? OK.

Comment (fabiangreffrath) [reply src/i_flmusic.c]: 👍

Comment (fabiangreffrath) [reply src/i_flmusic.c]: Let's say 10-1000.

**Review by fabiangreffrath — APPROVED** [review]

**Pull request merged**